custom_components/ryanair/translations/pt.json

```diff
@@ -0,0 +1,30 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Equipamento já configurado"
+        },
+        "error": {
+            "cannot_connect": "Falha na ligação",
+            "invalid_auth": "Autenticação invalida",
+            "invalid_mfa_code": "Codigo MFA errado",
```

Comment:
According to Google this translate to "Wrong MFA code". If that's the case, is "Código MFA inválido" a truer translation?

Reply:
wrong == Errado or Inválido is the same meaning

Reply:
Fair enough. I'm happy with this then.